src/lxc/tools/lxc-checkconfig.in

```diff
@@ -28,16 +28,27 @@ is_enabled() {
 
     is_set $1
     RES=$?
-
+    RET=1
     if [ $RES -eq 0 ]; then
-        $SETCOLOR_SUCCESS && echo "enabled" && $SETCOLOR_NORMAL
+        $SETCOLOR_SUCCESS && echo -n "enabled" && $SETCOLOR_NORMAL
+        RET=0
     else
         if [ ! -z "$mandatory" ] && [ "$mandatory" = yes ]; then
-            $SETCOLOR_FAILURE && echo "required" && $SETCOLOR_NORMAL
+            $SETCOLOR_FAILURE && echo -n "required" && $SETCOLOR_NORMAL
         else
-            $SETCOLOR_WARNING && echo "missing" && $SETCOLOR_NORMAL
+            $SETCOLOR_WARNING && echo -n "missing" && $SETCOLOR_NORMAL
         fi
     fi
+    return $RET
+}
+
+is_probed() {
+    lsmod | grep $1 > /dev/null
+    if [ $? -eq 0 ]; then
+        echo -n " probed"
```

Comment:
Hm, wouldn't it be better to say: "kernel module loaded" and "kernel module not loaded". "probed" "no probed" might confuse users.